sky/clouds/service_catalog/azure_catalog.py

```diff
@@ -138,7 +147,12 @@ def get_instance_type_for_accelerator(
     if zone is not None:
         with ux_utils.print_exception_no_traceback():
             raise ValueError('Azure does not support zones.')
-    return common.get_instance_type_for_accelerator_impl(df=_df,
+
+    # Filter out instance types that only contain a fractional of GPU.
+    df_filtered = _df.loc[~_df['InstanceType'].isin(_FILTERED_A10_INSTANCE_TYPES
+                                                   )]
```

Comment:
Instead of excluding the instances directly, can we print out some hints like the one when we specify `sky launch --gpus L4`:
```
Multiple AWS instances satisfy L4:1. The cheapest AWS(g6.xlarge, {'L4': 1}) is considered among:
I 08-27 06:09:54 optimizer.py:922] ['g6.xlarge', 'g6.2xlarge', 'g6.4xlarge', 'gr6.4xlarge', 'g6.8xlarge', 'gr6.8xlarge', 'g6.16xlarge'].
```

Reply:
This hint is used to print instances w/ same accelerator number. I'm thinking if we should do this to fractional GPUs..